### DISCUSSION TITLE
Nested include returns [Object] instead of object value

### DISCUSSION THREAD

**Discussion by spidertwin2** [prisma/prisma#7927] [Q&A] [answered]
Aloha :palm_tree:

When using `include` inside `select`, [Object] is returned instead of the linked object value.

I want to query `Post` to retrieve all `comments` and `author` `name` of each comment.

```ts
const data = await db.post.findUnique({
      where: {
        id: postId,
      },
      select: {
        comments: {
          include: {
            author: true,
          },
        },
      },
    })
```

```
{
  comments: [
    {
      id: 'ckqhptpzw0350nnem42lr6co6',
      body: 'First comment',
      postId: 'ckq26l7mq0032ssemyirds1ts',
      authorId: 1,
      author: [Object]  //why is the author object not retrieved?
    },
    {
      id: 'ckqhqne7o0411nnemf4wh1tdb',
      body: 'Second comment',
      postId: 'ckq26l7mq0032ssemyirds1ts',
      authorId: 1,
      author: [Object]
    }
  ]
}
```

```prisma
model User {
  id             Int            @id @default(autoincrement())
  name           String?
  comments       Comment[]      @relation("WrittenComments")
  posts          Post[]         @relation("WrittenPosts")
}

model Post {
  id        String   @id @default(cuid())
  title     String?
  body      String?

  comments Comment[] @relation("CommentsOnPost")
  author   User?     @relation("WrittenPosts", fields: [authorId], references: [id])
  authorId Int
}

model Comment {
  id        String   @id @default(cuid())
  body      String

  post           Post      @relation("CommentsOnPost", fields: [postId], references: [id])
  postId         String
  author         User      @relation("WrittenComments", fields: [authorId], references: [id])
  authorId       Int
}

```
Many thanks!

**Comment by ryands17** [comment, marked as answer]
Are you using `console.log` to view the value? If so try adding the response in `JSON.stringify`

**Reply by spidertwin2** [reply]
works!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
